pkg/cloudprovider/aws/ami.go

```diff
@@ -106,17 +106,9 @@ func (p *AMIProvider) getBottlerocketAlias(version string, instanceType cloudpro
 
 func (p *AMIProvider) getSSMQuery(ctx context.Context, constraints *v1alpha1.Constraints, instanceType cloudprovider.InstanceType, version string) string {
```

Comment:
Super minor nit, but I tend to put functions in the file in the order they're used. E.g., getSSMQuery above getAL2Alias, getBottlerocketAlias